doc/event-alarm-framework/event-alarm-framework.md

```diff
@@ -191,65 +180,58 @@ Application owners need to identify various conditions that would be of interest
-| 7.2.3 | Openconfig interface to pull alarms using following filters                     |                     |
+| 5.2.3 | Openconfig interface to pull alarms using following filters                     |                     |
 |       | - All (pull all events)                                                         |                     |
 |       | - Severity.                                                                     |                     |
 |       | - Recent alarms (eg., last 5 minutes, one hour, one day).                       |                     |
 |       | - Records between two timestamps, one timestamp and end, and beginning and a timestamp. |                     |
 |       | - All records between two Sequence Numbers (incl end and begin)                 |                     |
-| 7.2.4 | Openconfig interface  to acknowledge an alarm.                                  |                     |
-| 8     | CLI commands                                                                    |                     |
-| 8.1   | show alarm [ detail \| summary \| severity \| timestamp <from> <to> \| recent <5min\|1hr\|1day> \| sequence-number <from> <to> \| all]          |                    |
-| 8.2   | show event [ detail \| summary \| severity \| timestamp <from> <to> \| recent <5min\|1hr\|1day> \| sequence-number <from> <to>]                 |                     |
-| 8.3   | show event profile                                                              |                     |
-| 8.4   | alarm acknowledge <sequence id>                                                 |                     |
-| 8.5   | logging server <ip> [ log \| event ]                                             | default is 'log'   |
-| 8.6   | event profile   [ default \| name-of-file ]	                                  |                     |
-| 9     | gNMI subscription                                                               |                     |
-| 9.1   | Subscribe to openconfig Event container and Alarm container. All events and alarms published to gNMI subscribed clients. |                    |
-| 10    | Clear all events                                                                |                     |
-| 11    | Any change in open source should be aligned and upstream.                       |                     |
+| 5.2.4 | Openconfig interface  to acknowledge an alarm.                                  |                     |
+| 6     | CLI commands                                                                    |                     |
+| 6.1   | show alarm [ detail \| summary \| severity \| timestamp <from> <to> \| recent <5min\|1hr\|1day> \| sequence-number <from> <to> \| all]          |                    |
+| 6.2   | show event [ detail \| summary \| severity \| timestamp <from> <to> \| recent <5min\|1hr\|1day> \| sequence-number <from> <to>]                 |                     |
```

Comment:
In 202305 present codebase, do not see support for these CLIs (i.e. show alarm, show event).
[1] In their absence, how does one check eventD's cache-services contents? as to what events, alarms are received and stored in eventD's cache-services?
Is it there as part of some changeset not merged yet? or not there at all?

https://github.com/sonic-net/SONiC/pull/1064 has some unmerged PRs. Does this support reside there?
If so, what's the plan to commit, merge these pending PRs to sonic mainstream repositories?

Reply:
The present codebase does not have CLI support. It supports event publish via telemetry streaming.
The implementation for doc/event-alarm-framework/event-alarm-framework.md, is not merged. 
#1064 unmerged PRs have the support for CLIs. 
The current plan is to first get  this PR merged. @shyam77git Please review and approve this PR. 
Then rework changes in #1064 PRs to support these design updates and submit to community for coming release.